openshift/scripts/create-release-branch.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Usage: create-release-branch.sh v0.4.1 release-0.4
+
+release=$1
+target=$2
+
+git fetch upstream --tags
+git checkout -b "$target" "$release"
+
+git checkout master openshift OWNERS_ALIASES OWNERS Makefile
```

Comment:
shoulkd we add a little, `markdown` file, that explains the usage of these files?

Reply:
Good point, added a README.md to the release folder!